community/graphdb-api/src/main/java/org/neo4j/graphdb/Node.java

```diff
@@ -78,7 +78,9 @@
      * are attached to this node, an empty iterable will be returned.
      *
      * @return all relationships attached to this node
+     * @deprecated this method should return {@link ResourceIterable} instead
      */
+    @Deprecated
```

Comment:
I can remove introduced depreciations and leave only a note that it's actually a resource iterator returned that we rely on that and API will be changed in a future version? sounds good?